QRScannerSample/QRScannerSample/FlashButton.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  FlashButton.swift
+//  QRScannerSample
+//
+//  Created by daichiro on 2019/11/08.
+//  Copyright © 2019 mercari.com. All rights reserved.
+//
+
+import UIKit
+
+final class FlashButton: UIButton {
```

Comment:
Move FlashButton to QRScanner framework? If users want to use this Button, they can import it.

Reply:
I don't think so.
Flash handling is in framework but flash button is outside of framework, I think.
Users customize flash button as them likes.
So if `FlashButton` is not used, this may be capacity waste of binary 😕